JapaneseFilter/sections/specific.txt

```diff
@@ -43,9 +43,10 @@ hentai-covers.site,109cinemas.net,4545.to,bunshun.jp,eikaiwa.dmm.com,gizmodo.jp,
 !
 !
 !
-bintan.ula.cc##.ol2 > li.tListLI > center 
+bintan.ula.cc##.ol2 > li.tListLI > center
 doope.jp##.amazonbox
 jocr.jp##.article__ads
+simchange.jp##.code-block-18
```

Comment:
Better `##.recruit`

Reply:
Plus `||simchange.jp/wp-content/uploads/*_300-250.jpg` for future change.

Reply:
ok